src/pages/Options/Market/TradeCard/BiddingPhaseCard/TradeSide.tsx

```diff
@@ -53,21 +52,39 @@ const TradeSide: FC<TradeSideProps> = memo(
 		transKey,
 		currentPosition,
 		priceShift,
+		onMaxClick,
 	}) => {
 		const { t } = useTranslation();
-		const theme = useContext(ThemeContext);
 
 		const amountInputId = `${type}-${side}-amount`;
 		const priceInputId = `${type}-${side}-price`;
 
 		return (
 			<Container isActive={isActive} onClick={onClick}>
 				<Section>
+					<FormRow>
+						<SideIndication highlighted={isActive}>
+							{t(`options.common.${side}`)} <StyledSideIcon side={side} />
+						</SideIndication>
+					</FormRow>
 					<FormRow>
 						<FormControl>
-							<FormInputLabel htmlFor={amountInputId}>
-								{t(`${transKey}.amount-label`)}
-							</FormInputLabel>
+							<FormInputLabelRow>
+								<FormInputLabel htmlFor={amountInputId}>
+									{t(`${transKey}.amount-label`)}
+								</FormInputLabel>
+								<MaxButton
+									onClick={() => {
```

Comment:
Maybe something like 
```
onClick={ isActive ? () => onMaxClick() : undefined }
```

I'm not sure we really want to trigger the "onClick" here (it was for things like clicking the whole "Trade Side")